log/logrus/logrus_logger.go

```diff
@@ -10,18 +10,37 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type logrusLogger struct {
+type Logger struct {
 	logrus.FieldLogger
+	logrus.Level
```

Comment:
I just noticed that these fields are embedded. Do we want their methods added to the method set of `Logger`? Now that `Logger` is exported, that also means these fields are accessible outside this package. I don't think that is intended. It certainly opens the potential for other code to modify the level while other code is reading it in the `Log` method in a racy way.

I think these should have unexported field names.

What do you think @peterbourgon ?

Reply:
I agree.